doc/controls/AutoLayoutControl.md

```diff
@@ -44,4 +45,4 @@ PrimaryAlignment|AutoLayoutPrimaryAlignment|Either Auto, the child grid take onl
 CounterAlignment|AutoLayoutAlignment|Indicates where an element should be displayed on the counter axis of the parent orientation relative to the allocated layout slot of the parent element
 PrimaryLength|double| Set the grid size in parent orientation [Deprecated should Height and Width instead] 
 CounterLength|double|Sets the child size in parent counter orientation [Deprecated should Height and Width instead] 
-IsIndependentLayout  | bool | Make the child independent of the AutoLayout positioning. Should not be used with other Attached Properties. Reflect the Absolute Position option from Figma
\ No newline at end of file
+IsIndependentLayout  | bool | Make the child independent of the AutoLayout positioning. Should not be used with other Attached Properties. Reflect the Absolute Position option from Figma.
```

Comment:
We should either commit to ending each property description with a period or none of them, right now it's kind of all mixed

Reply:
and in addition, unify the spacing too: (pick one) space around `|` pipe OR no space around it.
the very least no double spacing:
```suggestion
IsIndependentLayout | bool | Make the child independent of the AutoLayout positioning. Should not be used with other Attached Properties. Reflect the Absolute Position option from Figma.
```

Reply:
thanks !